src/nodes.ts

```diff
@@ -15,6 +17,20 @@ export abstract class Node {
     this.range = [start, end];
     this.raw = raw;
   }
+
+  getChildren(): Array<Node> {
+    let children: Array<Node> = [];
+    for (let childField of this.getChildFields()) {
+      if (Array.isArray(childField)) {
+        children.push(...childField.filter<Node>((node): node is Node => node !== null));
```

Comment:
I'm surprised this `node is Node` assertion filter is needed. What does just `filter(node => node)` do?